README.md

```diff
@@ -40,17 +45,9 @@ To get a local copy up and running follow these simple example steps.
 
 ## Authors
 
-👤 **Author1**
```

Comment:
@KayLemba 
Kindly add an author name, and the details pertaining to the author such as Github handle, LinkedIn and email for a professional look

Reply:
Thank you for the review @koomekevo. Ill implement the required changes